README.md

```diff
@@ -54,7 +54,7 @@ Plugin versions and respective Velero/Kubevirt/CDI versions that are tested to b
 
 ## Install
 
-To install the plugin check current velero documentation https://velero.io/docs/v1.7/overview-plugins/.
+To install the plugin check current velero documentation https://velero.io/docs/main/overview-plugins/.
```

Comment:
Instead of adding the current development version I'd say it makes more sense to add the latest stable version (v1.15 in this case).

Reply:
@alromeros Or perhaps the latest version that corresponds to an OADP release, which would be v1.14 -- OADP 1.4 uses 1.14, and OADP 1.5 will use 1.16.

Reply:
Although maybe the OADP association isn't really relevant here, so 1.15 is probably fine too.

Reply:
Done, updated the PR.